connectors/glpi.py

```diff
@@ -0,0 +1,102 @@
+import logging
+from datetime import datetime
+
+from glpi_api import GLPI
+from sqlalchemy import desc, update
+
+from models.issue import Issue
+from utils.date import date_to_datetime
+from utils.timeit import timeit
+
+class GlpiConnector:
+
+    def __init__(self, project_id, session, config) -> None:
+        self.config = config
+        self.session = session
+        self.project_id = project_id
+        self.glpi = GLPI(url=self.config.glpi_base_url,
+                         apptoken=self.config.glpi_app_token,
+                         auth=(self.config.glpi_username, self.config.glpi_password))
```

Comment:
Both app token and username / password are needed ?

Reply:
Yes, token and username/password are needed to connect to GLPI and retrieve session token.